module2/index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <title>First Mobile Example</title>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="_css\Cook-a1.css">
+  <link rel="stylesheet" href="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.css" />
+  <script src="http://code.jquery.com/jquery-1.11.1.min.js"></script>
+  <script src="http://code.jquery.com/mobile/1.4.5/jquery.mobile-1.4.5.min.js"></script>
+
+</head>
+<body>
+<!-- Benjamin Cook
+     assignment?.html
+     INFO2439
+     Thoendel
+     9/4/2019
+-->
+<div id="homePage">
+  <div data-role="header">
+    <h1>Ben Cook</h1></div>
+    <h1>INFO 2439</h1>
+    <h2>Fall 2019</h2>
+  <div role="main" class="ui-content">
+      <ul data-role="listview" data-inset="true">
+        <li data-role="list-divider"><h3>Homework Assignments</h3></li>
```

Comment:
I don't think this is the index.html that is live on the server. I don't see a link for assignment 2 and you have one working (nice work) on the server.